### ISSUE TITLE
Integrate @dojo/web-editor and/or CodeSandbox into tutorials

### ISSUE THREAD

**Issue by vansimke** [dojo/dojo.io#93]

**Comment by vansimke** [comment]
TODO: Looks like we might have to add a step to the build process to generate the project.json files for each tutorial that will be shown in the editor.

**Comment by vansimke** [comment]
Putting this on hold pending design direction

**Comment by devpaul** [comment]
This might be blocked by #72

**Comment by vansimke** [comment]
Restarting this task to investigate changes required to update tutorials to [new design](https://app.atomic.io/d/LdFMCjK7dWDH?page=af7a612a-4102-4cdc-9a5f-0a58ed2e38fd)

**Comment by vansimke** [comment]
Create grunt tasks to:

1. find all demo projects
1. run npm install on each one
1. export project via `dojo export project` command
1. move all exported projects to /tutorials/assets directory 
1. remove `node_modules` folders from each demo project

Code can be found in [this branch](https://github.com/vansimke/dojo.io/tree/93-export-demo-projects).

**Comment by agubler** [comment]
All tutorials can be viewed on codesandbox and have had links added
